### PR TITLE
Switch to a newzealidar version that works on both Windows and Linux

### LINKED ISSUE #139: Change NZLidar Dependency to a single branch approach, managed by GRI
V0.1 and V0.2 are  very confusing ways to work on linux vs windows, fix these issues and merge the branches

### PR DESCRIPTION
**Pull request by LukeParky**
DESCRIPTION OF PR:

Closes #139 
Switches to a new NewZealidar version that is based on the merge of v0.1 and v0.2 so that they can run on Windows and Linux. This version also has the NewZealidar version locked down

## Developer Checklist
- [x] Make code change
- [x] Test on Linux
- [x] Test on Windows
- [x] Update documentation
  - [x] Readme
  - [x] Docstrings
  - [x] Comments
  - [x] Wiki

## Reviewer Checklist
- [ ] Check new code for code smells
- [ ] Check new tests
  - [ ] Ensure adequate coverage
  - [ ] Check for code smells within tests
- [ ] Check if documentation needs updating
  - [ ] Readme
  - [ ] Docstrings
  - [ ] Comments
  - [ ] Wiki

### CODE REVIEW

**Review by caseyli92 — APPROVED** [review]
Ensure it is fully tested before merging.

**Comment by LukeParky** [comment]
Doing one final test with `master` branch to ensure the results are the same and I will merge it if that comes out correct.

**Pull request merged**
